www/src/views/community/community-header.js

```diff
@@ -6,33 +6,33 @@ import Checkmark from "./check.svg"
 import Button from "../../components/button"
 import ArrowForwardIcon from "react-icons/lib/md/arrow-forward"
 
-const CommunityHeaderLink = ({ linkTo, children }) => (
-  <li
-    css={{
-      display: `flex`,
-      alignItems: `center`,
-      margin: 0,
-    }}
-  >
-    <Link
-      to={linkTo}
-      state={{ filter: `` }}
-      activeStyle={{
-        backgroundColor: colors.gatsby,
-        color: `white`,
-      }}
-      css={{
-        ...styles.communityHeaderLink,
-      }}
-    >
-      {children}
-    </Link>
-  </li>
-)
+// const CommunityHeaderLink = ({ linkTo, children }) => (
+//   <li
+//     css={{
+//       display: `flex`,
+//       alignItems: `center`,
+//       margin: 0,
+//     }}
+//   >
+//     <Link
+//       to={linkTo}
+//       state={{ filter: `` }}
+//       activeStyle={{
+//         backgroundColor: colors.gatsby,
+//         color: `white`,
+//       }}
+//       css={{
+//         ...styles.communityHeaderLink,
+//       }}
+//     >
+//       {children}
+//     </Link>
+//   </li>
+// )
 
 class CommunityHeader extends Component {
   render() {
-    const { forHire, hiring, submissionText } = this.props
+    const { /*forHire, hiring,*/ submissionText } = this.props
```

Comment:
decided to comment out "unused vars" as rest of the code that was using that is commented - presumably can be used in future?

Reply:
commented out code should be removed in general from master.